### PR TITLE
fix(command): return empty object detail if not found

### PR DESCRIPTION
**Pull request by adlerhurst**
- [x] I am happy with the code
- [x] Short description of the feature/issue is added in the pr description
- [x] PR is linked to the corresponding user story
- [x] Acceptance criteria are met
- [x] All open todos and follow ups are defined in a new ticket and justified
- [x] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [x] No debug or dead code
- [x] Critical parts are tested automatically
- [x] Where possible E2E tests are implemented
- [x] Documentation/examples are up-to-date
- [x] All non-functional requirements are met
- [x] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: internal/command/org_member.go [hunk 1]
```diff
@@ -145,7 +145,8 @@ func (c *Commands) RemoveOrgMember(ctx context.Context, orgID, userID string) (*
 		return nil, err
 	}
 	if errors.IsNotFound(err) {
-		return nil, nil
+		// empty response because we have no data that match the request
+		return &domain.ObjectDetails{}, nil
```
Comment (eliobischof) [inline internal/command/org_member.go:149]: Wouldn't an is not found error be more consistent?

**Review by eliobischof — CHANGES_REQUESTED** [review]
I think I saw the same thing on project or granted project member.

File: internal/command/org_member.go (hunk 1 above)
Comment (adlerhurst) [reply internal/command/org_member.go:149]: It would but break the API behaviour from previous versions

Comment (eliobischof) [reply internal/command/org_member.go:149]: Also, the test TestCommandSide_RemoveIAMMember/member_not_existing,_nil_result fails now @adlerhurst

Comment (eliobischof) [reply internal/command/org_member.go:149]: I changed expected return values in the tests

**Pull request merged**